Source/WebCore/Modules/mediastream/UserMediaRequest.cpp

```diff
@@ -139,7 +139,8 @@ void UserMediaRequest::start()
         break;
     }
 
-    PlatformMediaSessionManager::singleton().prepareToSendUserMediaPermissionRequest();
+    if (RefPtr page = document.protectedPage())
+        PlatformMediaSessionManager::singleton().prepareToSendUserMediaPermissionRequest(*page);
```

Comment:
Nit: it might be nice to update the name to something like `prepareToSendUserMediaPermissionRequestForPage`